src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -172,6 +208,22 @@ public IReadOnlyList<DataColumn> GetAllUsedDataColumns()
          return _dataColumnDTOCache.KeyValues.Where(x => x.Value.Used).Select(x => x.Key).ToList();
       }
 
+      public OutputMappings AllOutputMappings { get; set; }
+
+      public void OutputObservedDataLinkingChanged(bool isLinkedSimToData)
+      {
+         _isLinkedMappedOutputs = isLinkedSimToData;
```

Comment:
call the variable the same as the parameter => ``isLinkedMappedOutputs`

Reply:
done